src/lib/ui/app/SubscriptionPlan/BreakdownTable/breakdown.ts

```diff
@@ -225,7 +225,12 @@ export const BUSINESS_PLANS_BREAKDOWN = [
   },
 ]
 
-export const SubscriptionPlanBreakdown = {
+export type PlanFeatureBreakdown = boolean | string | number
+
+export const SubscriptionPlanBreakdown: Record<
+  string,
+  undefined | Record<string, PlanFeatureBreakdown>
+> = {
```

Comment:
Let's use following convention for defining types and generics:
- types are prefixed with `T`, e.g. `TPlanFeatureBreakdown`
- generics are prefixed with `G`, e.g. `<GComponent extends ComponentType>`

```suggestion
export type TPlanFeatureBreakdown = boolean | string | number

export const SubscriptionPlanBreakdown: Record<
  string,
  undefined | Record<string, TPlanFeatureBreakdown>
> = {
```